File: PassingDataObjC/PassingDataObjC/ViewController.m
//
//  ViewController.m
//  PassingDataObjC
//
//  Created by Field Employee on 3/26/20.
//  Copyright © 2020 Hugo Flores. All rights reserved.
//

#import "ViewController.h"
#import "ViewBController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    if ([segue.identifier  isEqual: @"toViewB"]) {
        ViewBController *controller = segue.destinationViewController;
        controller.passTextSegue = @"Hello World";
    }
}


@end

File: PassingDataSwift/PassingDataSwift/ViewController.swift
//
//  ViewController.swift
//  PassingDataSwift
//
//  Created by Field Employee on 3/26/20.
//  Copyright © 2020 Hugo Flores. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    /*
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        let segueDestination = segue.destination as! ViewBController
        segueDestination.passTextSegue = "Hello World"
    }
    */
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        if segue.identifier == "toViewB" {
            let controller = segue.destination as! ViewBController
            controller.passTextSegue = "Hello World"
        }
    }

}

